dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/engine/task/dispatcher/WorkerGroupDispatcher.java

```diff
@@ -84,23 +95,77 @@ public void run() {
     }
 
     private void doDispatchTask(ITaskExecutionRunnable taskExecutionRunnable) {
+        final int taskId = taskExecutionRunnable.getId();
```

Comment:
```suggestion
        final int taskExecutionRunnableId = taskExecutionRunnable.getId();
```

Avoid confuse with task definition id.

Reply:
> taskExecutionRunnableId

final int taskInstanceId = taskExecutionRunnable.getId();    is it better?

/**
     * Get the task instance id.
     * <p> Need to know the id might change since the task instance might be regenerated.
     */
    default int getId() {
        return getTaskInstance().getId();
    }

Reply:
It's better to use `taskExecutionRunnableId `, since the outer layer is unaware that this is the taskInstanceId.